<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>4</y>
      <width>361</width>
      <height>121</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QWidget" name="">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_x">
        <property name="minimum">
         <double>-10000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_op">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_y">
        <property name="minimum">
         <double>-10000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_eq">
        <property name="text">
         <string>=</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_ans">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_add">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_sub">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_mul">
        <property name="text">
         <string>*</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_div">
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_quit">
        <property name="text">
         <string>Quit</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>409</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_quit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>498</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
